spirit/category/models.py

```diff
@@ -27,6 +27,8 @@ class Category(models.Model):
 
     # topic_count = models.PositiveIntegerField(_("topic count"), default=0)
 
+    color = models.CharField(_("color"), max_length=7, null=True, blank=True)
```

Comment:
Oh, can you add a help text? something like `help_text=_("Title color in hex format (i.e: #1aafd0).")`